helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml

```diff
@@ -115,6 +115,8 @@ spec:
               {{- end }}
             - name: sys-path
               mountPath: /host-sys
+            - name: lib-modules
+              mountPath: /lib/modules
```

Comment:
could you please explain why need to do this?

Reply:
Mount /lib/modules will help process in container get some unusual kernel mod like broute & iptables_mangle.